app/helpers/assignment_index_helper.rb

```diff
@@ -7,17 +7,17 @@ def cal_day(day:)
 
   def cal_assignment(assignment:, day:)
     return if assignment.blank?
-    ass = 'assignment'
-    ass += '-user' if assignment.user == @current_user
+    assign = 'assignment'
+    assign += '-user' if assignment.user == @current_user
 
     if day == assignment.start_date && day == assignment.end_date
```

Comment:
Thanks! Good to know. I thought there could be a way to do this, but it might just be better to leave it as is.